test/parallel/test-child-process-exec-timeout.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+
+if (process.argv[2] === 'child') {
+  setTimeout(() => {
+    console.log('child stdout');
```

Comment:
Would you mind adding a comment mentioning that the `console.log()` calls are intentionally part of the test.